umh-core/pkg/config/nmapserviceconfig/generator.go

```
@@ -0,0 +1,85 @@
+	}
+
+	return configMap
+}
+
+// normalizeConfig applies nothing since Nmap has no default settings
+func normalizeConfig(raw map[string]any) map[string]any {
+	return raw
+}
+
+// templateData represents the data structure expected by the simplified Nmap YAML template
+type templateData struct {
```

golangci-lint (1.64.8)

type `templateData` is unused

(unused)